.github/workflows/release-nightly.yaml

```diff
@@ -0,0 +1,45 @@
+name: nightly release
+
+on: push
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
```

Comment:
I think we are using 1.19 now so should we have this at 1.19 or maybe use the version in `go.mod`?
```
go-version-file: './go.mod'
check-latest: true
```
If so, I suppose we should change `release.yaml` as well.

Reply:
Resolved in [`2fb6c78`](https://github.com/openfga/openfga/pull/508/commits/2fb6c78bb9703cf38416845979155b93c92bc761)